lib/orbital/api.rb

```diff
@@ -2,6 +2,8 @@ module Killbill #:nodoc:
   module Orbital #:nodoc:
     class PaymentPlugin < ::Killbill::Plugin::ActiveMerchant::PaymentPlugin
 
+      FIVE_MINUTES_AGO = (1 * 300)
```

Comment:
Yup, seems easier and simpler indeed. Could you just make it a configurable property?